### ISSUE TITLE
The "Allow Access" and "Cancel" buttons are not keyboard accessible, preventing users from interacting with them using keyboard input.

### ISSUE THREAD

**Issue by msftedad** [adobe/adobe.github.com#146]
### Expected Behaviour
Ensure the "Allow Access" and "Cancel" buttons are fully accessible and navigable using keyboard input to ensure usability for all users.

### Actual Behaviour
Users cannot interact with the "Allow Access" and "Cancel" buttons using keyboard input, hindering accessibility.

#### Steps to Reproduce

1. Open above URL https://codepen.io/Tester43/pen/QWXLNJJ in edge browser.
2. Navigate to the page controls using tab key.
3. Verify 'Allow Access' & 'Cancel' buttons are accessible using keyboard tab key or not.

https://github.com/adobe/adobe.github.com/assets/93735775/ba8de4e7-1acd-4b7e-a2ff-8411c6d94ca2

#### Platform and Version
Platform: Windows
Build: Windows 11 (Version 23H2 22631.3737)

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by the-piyushgoel** [comment]
is this issue still opened?

**Comment by pragya411** [comment]
Hi Team, This issue is still repro'ing. Please provide an update on this.

**Comment by vaishnavisapkale** [comment]
Hi 👋 I'd like to work on this issue. Could @msftedad you please assign it to me?

**Comment by msftedad** [comment]
Hi Team, this issue is still repro'ing. Please provide an update on this.
